Gateway/Request/RecurringDataBuilder.php

```diff
@@ -65,6 +66,8 @@ public function build(array $buildSubject): array
             $body = $this->vaultHelper->buildPaymentMethodRecurringData($storeId, $brand);
         } elseif ($method === PaymentMethods::ADYEN_ONE_CLICK) {
             $body = $this->adyenRequestsHelper->buildAdyenTokenizedPaymentRecurringData($storeId, $payment);
+        } elseif ($method === PaymentMethods::ADYEN_PAY_BY_LINK) {
+            return ['body' => $body];
```

Comment:
Can we refactor this so that it is aligned with the other `if conditions`? (The others assign a variable whereas this one returns)